internal/errgroup/group_test.go

```diff
@@ -511,102 +586,58 @@ func Test_group_Go(t *testing.T) {
 			if test.checkFunc == nil {
 				test.checkFunc = defaultCheckFunc
 			}
-			g := &group{
```

Comment:
I think it is better than `generator`, what do you think? (even it has lint error)

Reply:
I understand your opinion.
but I think we do not ignore lint error.
in this case, copying the mutex causes an error.
what do you think.

Reply:
maybe just remove line 521?

Reply:
ok.
but if the test use mutex or sync.once variable, I use a generator.
what do you think?

Reply:
using generator is the same. the difference of `generator` is that it is missing this statement `mu:  test.fields.mu,` , so the lint wouldn't report a error.

Reply:
:memo:
we may use generator to avoid coping `waitGroup` & `sync.Mutex`

so I close this comment.:pray: